tests/ipfwd/test_nhop_group.py

```diff
@@ -57,6 +62,7 @@ def program_routes(self):
                 ip_cmd = "{} {}".format(ip_route, ip_nhop_str)
                 fn.write(ip_cmd+ "\n")
 
+        fn.close()
```

Comment:
Is this related to the PR description?

Reply:
no, this is fixing existing mission option.